Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM quay.io/azavea/scala:2.10.6
+FROM openjdk:7-jre
```

Comment:
I started down the that path initially, but a bunch of stuff in the scripts that help manage Spark began to break down.

As far as Java 8 goes, my thinking was that we'd use that when we go from 1.6.x -> 2.x. Do you know if Java 8 is officially supported with Spark 1.6.x?